src/test/java/uk/gov/hmcts/reform/logging/layout/ReformLoggingLayoutTest.java

```diff
@@ -0,0 +1,141 @@
+package uk.gov.hmcts.reform.logging.layout;
+
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.joran.JoranConfigurator;
+import ch.qos.logback.core.joran.spi.JoranException;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.hmcts.reform.logging.exception.AbstractLoggingException;
+import uk.gov.hmcts.reform.logging.exception.AlertLevel;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InvalidClassException;
+import java.io.PrintStream;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(Parameterized.class)
+public class ReformLoggingLayoutTest {
+
+    private PrintStream old = System.out;
```

Comment:
typo

Reply:
where?

Reply:
why 'old'?

Reply:
it becomes old when replaced before tests. and reset after finish.